frontend/src/components/admin/masterListsPage/index.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
```

Comment:
I think we can use thie existing [admin](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop_3x/frontend/src/components/admin/Admin.js) to add the admin menu items for Configurations.
and we dont need to add all of them at once. we can just add the ones worked on

Reply:
Makes perfect sense